spec/middleware/exception_hunter/request_hunter_spec.rb

```diff
@@ -69,6 +70,40 @@ module ExceptionHunter
           error = Error.last
           expect(error.environment_data).not_to include(not_tracked_env)
         end
+
+        context 'when the exception was raised by a logged user' do
+          let!(:user) { User.create!(email: 'example@example.com', password: 'password') }
+          before do
+            allow(controller).to receive(:current_user).and_return(user)
+          end
+
+          it 'registers user data' do
+            begin
+              subject
+            rescue StandardError
+              nil
+            end
```

Comment:
We could also do
```suggestion
            subject rescue nil
```

I believe